### PR TITLE
DriveTest

### PR DESCRIPTION
**Pull request by andrewpmk**
DriveTest driver examination centres in Ontario

### CODE REVIEW

File: data/operators/office/government.json [hunk 1]
```diff
@@ -603,6 +603,19 @@
...
       }
     },
+    {
+      "displayName": "DriveTest",
+      "id": "drivetest-a69e09",
+      "locationSet": {
+        "include": ["ca-on.geojson"]
+      },
+      "tags": {
+        "name": "DriveTest",
```
Comment (AntiCompositeNumber) [inline data/operators/office/government.json:613]:
```suggestion
        "government": "transportation",
        "name": "DriveTest",
```

**Review by AntiCompositeNumber — COMMENTED** [review]
Looks like this is a pseudogovernmental office operated by Serco for the Ontario Ministry of Transportation. `office=government` is probably still the best tag for this, even though it has been privatized. I think this might actually fit better in the `brands/` tree rather than the `operators/` tree, which would allow us to specify `operator=Serco`.

**Comment by andrewpmk** [comment]
> Looks like this is a pseudogovernmental office operated by Serco for the Ontario Ministry of Transportation. `office=government` is probably still the best tag for this, even though it has been privatized. I think this might actually fit better in the `brands/` tree rather than the `operators/` tree, which would allow us to specify `operator=Serco`.

There isn't a file office/government.json in brands. Only in operator

**Comment by bhousel** [comment]
Thanks, it seems ok for this to be in the operator file if there isn't a brand file.

**Pull request merged**